components/inputs/test/input-date.vdiff.js

```diff
@@ -351,6 +351,68 @@ describe('d2l-input-date', () => {
 				await sendKeysElem(elem, 'press', 'Enter');
 				await expect(elem).to.be.golden();
 			});
+
+			it('click then click away', async() => {
+				await clickElem(elem);
+				await clickElem(document.body);
+				await expect(elem).to.be.golden();
+			});
+
+			it('open then close', async() => {
+				await sendKeysElem(elem, 'press', 'Enter');
+				await sendKeys('press', 'Escape');
+				await oneEvent(elem, 'd2l-tooltip-show');
+				await expect(elem).to.be.golden();
+			});
+		});
+
+		describe('required min-max revert', () => {
+
+			it('delete text input then blur', async() => {
```

Comment:
I want to confirm what should happen here with Jeff once he's back since this is also reverting and not telling the user. I've kept the behaviour as-is for right now.